CHANGES.md

```diff
@@ -10,6 +10,9 @@
 
 * Update `parking_lot` to 0.11 [#404]
 
+* Remove unnecessary `PhantomData` field from `Request` making it `Send + Sync`
+  regardless if `Request`'s type-argument is `Send` or `Sync` [#407]
```

Comment:
the PR link needs an entry below, it doesnt auto-link unfortunately

Reply:
Oh wow, some proper tunnel vision on my part here. Sorry. Fixed.